config/stack_gaea.yaml

```diff
@@ -147,6 +147,11 @@ g2:
   version: v3.4.1
   install_as: 3.4.1
 
+g2c:
+  build: YES
```

Comment:
Change this to NO please and we are ready to merge. Thanks!

Reply:
Done - let's someone merge it.

Reply:
done.